install.sh

```diff
@@ -643,7 +680,9 @@ create_systemd_service_file() {
 [Unit]
 Description=Lightweight Kubernetes
 Documentation=https://k3s.io
+After=network-online.target
 Wants=network-online.target
+After=network-online.target
```

Comment:
This line got duped some time ago, can we remove it now?

Reply:
I don’t want to fix anything in this PR, I want it to be just a straight pull of existing commits from master. I think there's a PR to fix this on master though, lets see if we can get that in.